packages/reporter/src/controllers/TelepathyReporterController.ts

```diff
@@ -0,0 +1,92 @@
+import axios from "axios"
+import { Chain, hexToNumber } from "viem"
+import { Mutex } from "async-mutex"
+
+import LightClientContractABI from "../ABIs/TelepathyContractABI.json" assert { type: "json" }
+import AdapterContractABI from "../ABIs/TelepathyAdapterABI.json" assert { type: "json" }
+import BaseController from "./BaseController.js"
+
+import { BaseControllerConfigs } from "./BaseController.js"
+
+// TODO: Update to comply with Hashi v0.2.0
```

Comment:
if it's not compatible with Hashi 0.2.0 please delete it. Also others in case there were

Reply:
Deleted in [7d03cc0](https://github.com/gnosis/hashi/pull/66/commits/7d03cc0f34c422357af961057a4931a7ba842c8f)